File: spring-cloud-starter/src/main/java/com/jmsoftware/maf/springcloudstarter/quartz/entity/GetQuartzJobConfigurationPageListPayload.java
package com.jmsoftware.maf.springcloudstarter.quartz.entity;

import com.jmsoftware.maf.common.bean.PaginationBase;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * <h1>GetQuartzJobConfigurationPageListPayload</h1>
 * <p>
 * Change description here.
 *
 * @author Johnny Miller (鍾俊), email: johnnysviva@outlook.com, 9/27/21 12:07 AM
 **/
@Data
@EqualsAndHashCode(callSuper = true)
public class GetQuartzJobConfigurationPageListPayload extends PaginationBase {
    private String serviceName;
}

File: spring-cloud-starter/src/main/kotlin/com/jmsoftware/maf/springcloudstarter/quartz/entity/GetQuartzJobConfigurationPageListPayload.kt
package com.jmsoftware.maf.springcloudstarter.quartz.entity

import com.jmsoftware.maf.common.bean.PaginationBase

/**
 * # GetQuartzJobConfigurationPageListPayload
 *
 * Change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/14/22 10:38 PM
 */
class GetQuartzJobConfigurationPageListPayload : PaginationBase() {
    var serviceName: String? = null
}
